op-node/rollup/driver/driver.go

```diff
@@ -82,8 +81,18 @@ type Network interface {
 	PublishL2Payload(ctx context.Context, payload *eth.ExecutionPayload) error
 }
 
+type AltSync interface {
+	// RequestL2Range informs the sync source that the given range of L2 blocks is missing,
```

Comment:
We probably should be clear about whether this range is inclusive or not as well.